modules/ROOT/pages/api-search.adoc

```diff
@@ -0,0 +1,38 @@
+= Search Inside Your API Specification Dependencies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Because Anypoint Studio 7.4 enables you to import your API specifications from Exchange or Maven as dependencies of your project, those dependencies no longer locate as resources inside the `src/resources/api/` directory, but as ZIP file dependencies in your project's classpath. This change makes your API dependencies not searchable through the File Search option in Studio. To enable you to search for specific text inside your API dependencies, Studio introduces API Search.
+
+You can access the API Search menu from the taskbar at the top of the Anypoint Studio display, selecting *Search* > *Search...*, or by using the `control + H` keyboard shortcut.
+
+image::api-search.png[]
+
+. Type the text that you want to search inside your API dependencies. +
+* Use `\*` to search for any string. For example, `*-api-*` searches for strings containing `-api-`.
+* Use `?` to allow any character after a specific pattern. For example, `customer?` searches for both `customer` and `customers`.
```

Comment:
```suggestion
* Append `?` to your search string to allow any character at the end of the string. 
For example, `customer?` searches for both `customer` and `customers`.
```

Reply:
Would it also search for `customer-related`? Just curious as to the limits of what or how many characters can be appended.